pkg/runtime/executor.go

```diff
@@ -264,7 +272,9 @@ func (e *Executor) findViolations(results *Output) error {
 
 	for _, engine := range e.policyEngines {
 		go func(eng policy.Engine) {
-			output, err := eng.Evaluate(policy.EngineInput{InputData: &results.ResourceConfig})
+			// create a regodata pre scan filter
+			preScanFilter := filters.RegoDataFilter{}
```

Comment:
This is good for now. Maybe we can implement a factory pattern for filters later ;)

Reply:
if we're not using this variable except for passing it in line 277, maybe we skip this line.